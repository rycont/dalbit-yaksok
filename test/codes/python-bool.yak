a = True
b = False

if a:
    'a는 참' 보여주기

if b:
    'b는 참' 보여주기
else:
    'b는 거짓' 보여주기
