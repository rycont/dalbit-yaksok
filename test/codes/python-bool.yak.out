a는 참
b는 거짓
